givitsite/gatherer/agora.py

```diff
@@ -0,0 +1,137 @@
+                   'Beer Sheva': 'באר שבע', 'Haifa': 'חיפה', 'Modiin': 'מודיעין', 'Hasharon': 'הוד השרון'}
+
+    items_list = get_items_list()
+    i = 0
+    for i in range(len(items_list)):
+        iseek = items_list[i][2]
+        my_soup = Agora_Getrequest(iseek)
+        url_list = find_furniture(
+            my_soup, region_dict.get(items_list[i][3]), iseek_dict.get(items_list[i][2]))
+        for item in range(len(url_list)):
+            items_list[i].append(url_list[item])
+        url_list = []
+    return items_list
+
+
+def get_items_list():
+    """
+    this function open our wishlist and return item_list such that every item in this list is a list with all the
+    information about.
+    Returns:
+    item_list (list of lists): list of all items in our wishlist.
+    """
+    wish_list = ItemRequest.objects.all()
+    items_list = [list(item.values()) for item in wish_list.values()]
+    return items_list
+
+
+def get_iseek(item):
+    """
+    this function gets name item in hebrew and return the iseek code in agora website.
+    Parameters:
+    item (string): item is the name of our item in hebrew.
+    Returns:
+    iseek (string): represent the code of our current item in the url, its in a dict such that iseek is the value of key item.
+    """
+    iseek_dict = {'ארון': '20009', 'מיטה': '20016', 'כיסא': '20008',
+                  'מקרר': '10006', 'מכונת כביסה': '10029', 'ספה': '20017'}
+    return iseek_dict[item]
+
+
+def Agora_Getrequest(iseek):
+    """
+    this function gets iseek which allow us to nevigate to the specific url and return beatifulsoup object.
+    Parameters:
+    iseek (string): iseek is the category number of the product that we need for the url.
+    Returns:
+    BeautifulSoup object: which represents the document as a nested data structure.
+    """
+    url = 'https://www.agora.co.il/toGet.asp?'
+    headers = {}
+    headers['User-Agent'] = 'Mozilla/5.0'
+    values = {'searchType': 'subCategory', 'dealType': '1', 'iseek': iseek}
+    data = urllib.parse.urlencode(values)
+    new_url = url + data
+    data = data.encode('utf-8')
+    req = urllib.request.Request(new_url, data, headers)
+    webpage = urllib.request.urlopen(req).read()
+    soup = BeautifulSoup(webpage, 'html.parser', from_encoding="utf-8")
+    return soup
+
+
+def find_furniture(soup, area, name):
+    """
+    this function gets soup, area and name of the product we are looking for
+    and return all the rows that found on the website as url_list.
+    Parameters:
+    soup (beautifulsoup object): which represents the document as a nested data structure.
+    area (string): the area that we want to pick the product from.
+    name (string): the name of the product we search for.
+    Returns:
+    url_list (list): list of all url that match to what we search.
+    """
+    url_list = []
+    t = soup.findAll('tbody', class_='objectGroup')
+    for i in range(len(t)):
```

Comment:
variable names like t, c1, c2 make it really hard to understand what the code is doing.

How about `item_table_body`, `area_column`, etc.